doc/nrf/gs_programming.rst

```diff
@@ -58,18 +58,31 @@ You can download it from the following links:
 #. Click **OK** to import the project into SES. You can now work with the
    project in the IDE.
 
-#. To build your project, select **Build -> Build zephyr/zephyr.elf**.
+#. Build and flash your project.
+   The required steps differ depending on if you build a single application or a multi-image project.
 
-    .. note::
+   To build a single application:
+
+      a. Select **Build -> Build zephyr/zephyr.elf**.
+      #. When the build completes, you can flash the sample to a connected board.
+         To do this, select **Target -> Download zephyr/zephyr.elf**.
+
+      .. note::
 	   Alternatively, choose the **Build and Debug** option.
 	   **Build and Debug** will build the application and flash it when
 	   the build completes.
 
-6. Once the build completes, you can flash the sample to a connected board.
-To do this, select **Target -> Download zephyr/zephyr.elf**.
+   To build a multi-image project:
+
+       a. Select **Build -> Build Solution**.
+       #. When the multi-image build completes, you can flash the sample to a connected board.
+          To do this, select **Target -> Download File -> Download Intel Hex File**.
+          Navigate to the ``zephyr`` folder in the build directory and select ``merged.hex``.
```

Comment:
Is that what you meant? If not, can you send me a screenshot of that window?

Reply:
Will do :)

Reply:
Done